src/get_opening_hours.ts

```diff
@@ -38,6 +46,62 @@ function toHourGroup(hourRange: HourRange): HourGroup | null {
   return { from: fromWithLeadingZero, to: toWithLeadingZero };
 }
 
+// Pattern to match absolute date like "Jan 26" or "Apr 13"
+const absoluteDatePattern = /^([A-Z][a-z]{2})\s+(\d+)$/;
+
+// Check if a day group string starts with an absolute date pattern
+function startsWithAbsoluteDate(dayGroup: string): boolean {
+  return /^[A-Z][a-z]{2}\s+\d/.test(dayGroup);
```

Comment:
Can't you reuse the pattern declared above? Or somehow modify it not to repeat yourself?